src/store.js

```diff
@@ -126,6 +131,20 @@ const actions = {
 
     commit('deleteKey', key);
     return { success: true, data };
+  },
+
+  populateRecentlyOpened({ commit }) {
+    const storedData = window.localStorage.getItem('recentlyOpened');
+    if (storedData) {
+      commit('setRecentlyOpenedPaths', JSON.parse(storedData));
+    }
+  },
+
+  addToRecentlyOpened({ commit, state }, path) {
+    const newPaths = state.recentlyOpenedPaths.filter(p => p != path);
+    newPaths.unshift(path);
```

Comment:
```suggestion
    const newPaths = [path, ...state.recentlyOpenedPaths.filter(p => p != path)];
```

How about something like this? 😅

Reply:
updated